### PR TITLE
CMake update after Eyescale/CMake#341

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -83,52 +90,37 @@ else()
...
-  add_executable(SimpleSimulator SimpleSimulator.cpp)
-  target_link_libraries(SimpleSimulator TUIO ${GLUT_LIBRARIES}
-                                        ${OPENGL_LIBRARIES} ${SDL_LIBRARY})
-  install(TARGETS TuioDemo SimpleSimulator
-          DESTINATION bin COMPONENT apps)
+  set(SIMPLESIMULATOR_SOURCES SimpleSimulator.cpp)
+  set(SIMPLESIMULATOR_LINK_LIBRARIES
+    TUIO ${GLUT_LIBRARIES} ${OPENGL_LIBRARIES} ${SDL_LIBRARY})
+
+  common_application(SimpleSimulator SimpleSimulator.cpp)
```
Comment (rdumusc) [inline CMakeLists.txt]: SimpleSimulator.cpp already in SIMPLESIMULATOR_SOURCES

File: CMakeLists.txt [hunk 2]
```diff
@@ -83,52 +90,37 @@ else()
...
-        ARCHIVE DESTINATION lib COMPONENT dev
-        RUNTIME DESTINATION bin COMPONENT lib
-        LIBRARY DESTINATION lib COMPONENT lib)
-
-add_executable(TuioDump TuioDump.cpp)
-target_link_libraries(TuioDump TUIO)
-install(TARGETS TuioDump DESTINATION bin COMPONENT apps)
+set(TUIODUMP_SOURCES TuioDump.cpp)
+set(TUIODUMP_LINK_LIBRARIES TUIO)
+common_application(TuioDump TuioDump.cpp)
```
Comment (rdumusc) [inline CMakeLists.txt]: TuioDump.cpp already in TUIODUMP_SOURCES

File: CMakeLists.txt (hunk 1 above)
Comment (tribal-tec) [inline CMakeLists.txt]: Good catch

File: CMakeLists.txt (hunk 2 above)
Comment (tribal-tec) [inline CMakeLists.txt]: thx

**Comment by tribal-tec** [comment]
Updated

**Comment by rdumusc** [comment]
+1

**Pull request merged**